### DISCUSSION TITLE
Account not being created correctly on API endpoint

### DISCUSSION THREAD

**Discussion by gilney-canaltelecom** [chatwoot/chatwoot#8452] [Chatwoot APIs]
### Describe the bug

When you create an account using the `/platform/api/v1/accounts` endpoint, the account is created, the features passed are setted, but then the account isn't saved again to save the features editions.

This happens on [this function](https://github.com/chatwoot/chatwoot/blob/dd05482cb0f728918f9c11186db67704c966c913/app/controllers/platform/api/v1/accounts_controller.rb#L4).

### To Reproduce

1. Send a POST request to the endpoint `/platform/api/v1/accounts` with the following body:
```JSON
{
  "name": "Test",
  "locale": "en_US",
  "features": {
        "agent_bots": true
  }
}
```
2. Notice that the return has the "agent_bots" setted to true.
3. Send a GET request to the endpoint `/platform/api/v1/accounts/:account_id` and notice that there will be no "agent_bots" in the response.

### Expected behavior

The features being passed to the `/platform/api/v1/accounts` endpoint are correctly saved.

### Environment

Linux VM

### Cloud Provider

None

### Platform

None

**Comment by yasirazgar** [comment]
@gilney-canaltelecom I would like to work on this.

**Comment by gilney-canaltelecom** [comment]
@yasirazgar If you can, please feel free to do it ^^ (I'm just a user reporting the bug, tho 👍)

**Comment by gilney-canaltelecom** [comment]
hum... why did this turned into a discussion when it seems to be only 1 line of code missing? o.O
